src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+
```

Comment:
this file should not exist.  at each checkout, your file in the project will be overwritten with an empty one.